auth/src/main/java/com/firebase/ui/auth/ui/email/RegisterEmailFragment.java

```diff
@@ -86,6 +92,16 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         }
     }
 
+    @Override
+    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        if (!mPasswordInput.isHintAnimationEnabled()) {
```

Comment:
In our app there is a delay because of presenter so it doesn't look good. In the library it's fast, so not needed, remove this.